src/App.js

```diff
@@ -1,24 +1,48 @@
 import React from 'react';
-import logo from './logo.svg';
+import { Provider } from 'react-redux';
+import { createStore, combineReducers, applyMiddleware } from 'redux';
+import createSagaMiddleware from 'redux-saga';
 import './App.css';
+import Routes from './routes/Routes';
+import rootSaga from './sagas/rootSaga';
+import loginReducer from './reducers/LoginReducer';
+import registrationReducer from './reducers/registrationReducer';
+import cartReducer from './reducers/cartReducer';
+import footerElementListReducer from './reducers/footerReducer';
+import productListReducer from './reducers/productListReducer';
+import userprofileReducer from './reducers/userprofileReducer';
+import userprofileupdateReducer from './reducers/userprofileupdateReducer';
+import addProductReducer from './reducers/addProductReducer';
+import updateProductReducer from './reducers/updateProductReducer';
+import searbarReducer from './reducers/searchbarReducer';
+import alertReducer from './reducers/alertReducer';
+
+const sagaMiddleWare = createSagaMiddleware();
+const store = createStore(
+  combineReducers({
```

Comment:
Can we move combine reducer to a separate file, just to follow good practice.

Reply:
moved to src/reducers/combinedReducer.js